src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java

```diff
@@ -292,6 +295,27 @@ private static int stopGame(CommandContext<ServerCommandSource> context) throws
 
         PlayerSet playerSet = gameSpace.getPlayers().copy();
```

Comment:
This doesn't need to copy anymore since it doesn't do anything besides check size